### ISSUE TITLE
HTML within Markdown does not get parsed correctly

### ISSUE THREAD

**Issue by naehrstoff** [interactivethings/catalog#10]
```
# Example Markdown

This <a href="#">link</a> is not converted correctly.

This &amp; and this &rarr; are not converted correctly.
```

**Comment by herrstucki** [comment]
Should get addressed in #37, although I'm not convinced that this is a priority.

**Comment by herrstucki** [comment]
I'm closing this, as I don't think we really need this anymore. Special characters like `&` are rendered correctly without needing to use html entities.
